src/components/ContactCard/ContactForm.jsx

```diff
@@ -77,7 +77,62 @@ const initialFieldValues = fields.reduce((initialValues, { name, isArray }) => {
   return initialValues
 }, {})
 
```

Comment:
file to rename ? I like having component name === file name :-°

Reply:
in this case, the default export is the "connected" `ContactForm`, the Dumb one is for tests (and to avoid to have both default export and named export with the name of the file). I should/will use export / as for this actually